docs/src/main/asciidoc/README.adoc

```diff
@@ -0,0 +1,238 @@
+:repo_url: https://github.com/marcingrzejszczak/database-update
+
+= Zero Downtime Deployment Database Issues
+
+{repo_url}
+
+This article will explain in more depth how to tackle issues related to database compatibility with Spring Boot and
+database versioning tooling. We will present what can happen with your production applications if you try to perform
+such a deployment unprepared. We will also walk through the lifecycle of an application in order to perform zero
```

Comment:
also->then?

Suggest something like this: "We will then wall through the steps in the lifecycle of an application that are necessary in order to have zero downtime."